l1-contracts/scripts/utils.ts

```diff
@@ -215,6 +216,7 @@ export function deployedAddressesFromEnv(): DeployedAddresses {
     Create2Factory: getAddressFromEnv("CONTRACTS_CREATE2_FACTORY_ADDR"),
     ValidatorTimeLock: getAddressFromEnv("CONTRACTS_VALIDATOR_TIMELOCK_ADDR"),
     Governance: getAddressFromEnv("CONTRACTS_GOVERNANCE_ADDR"),
+    BlobVersionedHash: getAddressFromEnv("CONTRACTS_BLOB_VERSIONED_HASH_ADDR"),
```

Comment:
Probably better to rename it to something like "BlobVersionedHashRetriever" or something along this lines? I understand that the temporary contract is already named as `BlobVersionedHash`, but I would rename it too so that it is clear that it retrieves those